src/propagators.jl

```diff
@@ -0,0 +1,145 @@
+using QuBase
```

Comment:
As the modules are loaded already in QuDynamics.jl, I removed the lines.
The docs I rebuilt them, so they have been replaced.